src/ripple/basics/base_uint.h

```diff
@@ -320,6 +320,7 @@ class base_uint
     {
         unsigned char* pOut  = begin ();
 
+        assert(sizeof(pn) == bytes);
```

Comment:
Already tested on Windows. Doesn't work as a `static_assert`, which is why the `assert` is seemingly stuck into a random function.